OracleWebLogic/samples/12213-domain-home-in-image/container-scripts/startManagedServer.sh

```diff
@@ -39,11 +39,14 @@ if [ -z "${PASS}" ]; then
 fi
 
 #Set Java Options
-JAVA_OPTIONS=`awk '{print $1}' ${SEC_PROPERTIES_FILE} | grep ^JAVA_OPTIONS= | cut -d "=" -f2`
-if [ -z "${JAVA_OPTIONS}" ]; then 
-   JAVA_OPTIONS="-Dweblogic.StdoutDebugEnabled=false"
+# Use the Env JAVA_OPTIONS if it's been set
+if [ -z "$JAVA_OPTIONS" ]; then
```

Comment:
IMHO it would be a better choice to have this logic augment the command line `JAVA_OPTIONS` value with the options specified in the properties file rather than simply have the command line override the properties setting?

I'm thinking of the case where the user does place some expected `JAVA_OPTIONS` in the properties file but then wants to supplement those with temporary debug flags for WebLogic or even JDK network SSL debug.

In the `README.md` the updated text does say override but we should be clear that it's a replacement or that the value set on the command line will augment as I'm suggesting that we do here...